### PR TITLE
Linux support for tfont (Qt), from pr #51

### PR DESCRIPTION
**Pull request by ideasman42**
Credit goes to François Revol for this work.

### CODE REVIEW

File: toonz/sources/common/tvrender/tfont_qt.cpp [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+struct TFont::Impl {
+  bool m_hasKerning;
+  int m_hasVertical;
+  QFont m_font;
+  // XXX:cache a QFontMetrics m_metrics; ?
+  // XXX:cache a QRawFont m_raw; ?
+
+  //  KerningPairs m_kerningPairs;
+
+#if 0
```
Comment (kogaki) [inline toonz/sources/common/tvrender/tfont_qt.cpp]: I feel just remove is better than comment out

**Comment by kogaki** [comment]
The entire code is written carefully and comprehensive. It's really great work. LGTM

File: toonz/sources/common/tvrender/tfont_qt.cpp (hunk 1 above)
Comment (ideasman42) [inline toonz/sources/common/tvrender/tfont_qt.cpp]: done

**Comment by kogaki** [comment]
Jenkins, test this

**Comment by kogaki** [comment]
Merged. But make this to work, we need to carefully replace existing (and legacy) `tfont` implementations with this brand new Qt implementation. 
Anyway, thanks for making great advance.

**Pull request merged**